FXThemes-samples/pom.xml

```diff
@@ -101,12 +150,14 @@
               <noHeaderFiles>true</noHeaderFiles>
               <options>
                 <!-- So that changing the background of a native Window works well (at least on some machines using win11) -->
-                <option>-Dprism.forceUploadingPainter=true</option>
+                <option>-Dprism.forceUploadingPainter=${forceUploadingPainter}</option>
                 <!-- For getting the native window handle (WindowUtils.getNativeHandleForStage method) -->
                 <option>--add-opens</option>
                 <option>javafx.graphics/javafx.stage=com.pixelduke.fxthemes</option>
                 <option>--add-exports</option>
                 <option>javafx.graphics/com.sun.javafx.tk.quantum=com.pixelduke.fxthemes</option>
+                <option>--add-exports</option>
+                <option>javafx.graphics/com.sun.glass.ui=com.pixelduke.fxthemes</option>
```

Comment:
Good call. This is removed because it's no longer complaining about access via reflection. 

I've removed and refactored WindowsUtils. The used to make a call to a package in `com.sun.glass.ui` using reflection. Now it calls the similar function `getNativeHandleOfStage()`